### PR TITLE
add explicit netstandard2.0

### PR DESCRIPTION
**Pull request by SimonCropp**
i reverted the last change.  (https://github.com/Fody/PropertyChanged/commit/8217a28aa16dd67140e5d97469e84ce8db7991a4) and just added two more TFW

### CODE REVIEW

**Comment by SimonCropp** [comment]
fixes #668

File: PropertyChanged/PropertyChanged.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>net40;net472;netstandard1.0;netstandard2.0</TargetFrameworks>
```
Comment (ltrzesniewski) [inline PropertyChanged/PropertyChanged.csproj:4]: I don't think `net472` is necessary... Is there a reason to add it?

Comment (SimonCropp) [reply PropertyChanged/PropertyChanged.csproj]: fixed

**Comment by ltrzesniewski** [comment]
LGTM

image

**Review by tom-englert — APPROVED** [review]
👍

**Pull request merged**